libmscore/types.h

```diff
@@ -285,6 +285,34 @@ enum class AccidentalType : char {
       NINE_TWELFTH_FLAT,
       NINE_TWELFTH_SHARP,
 
+      // (Spartan) Sagittal
+      S1,
+      S2,
+      S3,
+      S4,
+      S5,
+      S6,
+      S7,
+      S8,
+      S9,
+      S10,
+      S11,
+      S12,
+      S13,
+      S14,
+      S15,
+      S16,
+      S17,
+      S18,
+      S19,
+      S20,
+      S21,
+      S22,
+      S23,
+      S24,
+      S25,
+      S26,
```

Comment:
Need to come up with more useful names here I guess

Reply:
Would these work?:


SAGITTAL_5V7KU
--
SAGITTAL_5V7KD
SAGITTAL_5CU
SAGITTAL_5CD
SAGITTAL_7CU
SAGITTAL_7CD
SAGITTAL_25SDU
SAGITTAL_25SDD
SAGITTAL_35MDU
SAGITTAL_35MDD
SAGITTAL_11MDU
SAGITTAL_11MDD
SAGITTAL_11LDU
SAGITTAL_11LDD
SAGITTAL_35LDU
SAGITTAL_35LDD
SAGITTAL_SHARP25SD
SAGITTAL_FLAT25SU
SAGITTAL_SHARP7CD
SAGITTAL_FLAT7CU
SAGITTAL_SHARP5CD
SAGITTAL_FLAT5CU
SAGITTAL_SHARP5V7KD
SAGITTAL_FLAT5V7KU
SAGITTAL_SHARP
SAGITTAL_FLAT

Reply:
yep, thanks.